Formula/curl.rb

```diff
@@ -1,11 +1,10 @@
 class Curl < Formula
   desc "Get a file from an HTTP, HTTPS or FTP server"
   homepage "https://curl.se"
-  url "https://curl.se/download/curl-7.83.1.tar.bz2"
-  mirror "https://github.com/curl/curl/releases/download/curl-7_83_1/curl-7.83.1.tar.bz2"
-  mirror "http://fresh-center.net/linux/www/curl-7.83.1.tar.bz2"
-  mirror "http://fresh-center.net/linux/www/legacy/curl-7.83.1.tar.bz2"
-  sha256 "f539a36fb44a8260ec5d977e4e0dbdd2eee29ed90fcedaa9bc3c9f78a113bff0"
+  url "https://curl.se/download/curl-7.83.1.tar.xz"
+  mirror "https://github.com/curl/curl/releases/download/curl-7_83_1/curl-7.83.1.tar.xz"
+  mirror "http://fresh-center.net/linux/www/curl-7.83.1.tar.xz"
```

Comment:
Some formulae have HTTP mirrors in order to enable building them from source with an old version of `curl`. Using a `.xz` source tarball means that `xz` becomes a build dependency for these formulae, which we don't want.

Reply:
Running `brew info` on the `curl` formula with the `.xz` tarball doesn't mention an `xz` dependency anywhere. Am I missing something?

Reply:
Yes, brew will download xz if it's needed to unpack a source.

Reply:
Alright, I'll remove all `.xz` usages I introduced in this PR.